### ISSUE TITLE
Breaking changes to API

### ISSUE THREAD

**Issue by sachinruk** [tensorflow/tensorflow#7836]
`tf.nn.sampled_softmax_loss` and `tf.nn.nce_loss` have both changed their API such that you need to switch the `inputs, labels` to `labels, inputs` parameters. You might want to put a note here (under breaking changes): https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md

Incase you want to check here is the API of the two versions of tf:
https://www.tensorflow.org/versions/r0.12/api_docs/python/
https://www.tensorflow.org/api_docs/python/tf/nn/nce_loss

Although I do find it odd that tf would simply change the order of these inputs.

**Comment by gunan** [comment]
@aselle Was this API change intentional?
We might have missed this in our list for 1.0.

**Comment by aselle** [comment]
I think this was intentional and done by @martinwicke

**Comment by martinwicke** [comment]
This was intentional and done by @alextp. If it's not in the 1.0 release notes that was an oversight, these functions should have been included in the list of functions for which we switched arg order (such as softmax_cross_entropy_with_logits). We can add it to the 1.0 release note section of the release notes.

**Comment by gunan** [comment]
Release.dm updated. Closing this issue.
